### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by dkrammer**
Changed some activity names, added unsubscribe functionality

### CODE REVIEW

**Review by edenemilynan — COMMENTED** [review]
Can you tell me why in one file the custom toolbar import is deleted? And in a couple it looks like DB imports are axed? Just want to make sure that was intentional, or that they were unused so automatically removed :)

**Comment by dkrammer** [comment]
> 
> 
> Can you tell me why in one file the custom toolbar import is deleted? And in a couple it looks like DB imports are axed? Just want to make sure that was intentional, or that they were unused so automatically removed :)

Oh yeah, those were all automatic, I didn't touch those at all 👍

File: project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSubbedExperimentActivity.java [hunk 1]
```diff
@@ -59,9 +63,13 @@ protected void onCreate(Bundle passedData) {
         TextView region = findViewById(R.id.region_text_view);
         experimentManager.FB_UpdateExperimentTextViews(expID,expName,expDesc,expType,minTrials,region);
         trialManager.FB_FetchPublishedTrialCount(exp,currTrials);
+<<<<<<< HEAD
```
Comment (edenemilynan) [inline project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSubbedExperimentActivity.java]: OOPS, this merge conflict resolution doesn't look right 😬

Comment (dkrammer) [reply project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSubbedExperimentActivity.java]: Okay in my defense Android Studio said there was a conflict in that file but refused to tell me what it was, hold the phone

File: project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSearchedExperimentActivity.java [hunk 2]
```diff
@@ -70,10 +70,19 @@ protected void onCreate(Bundle passedData) {
         experimentManager.FB_UpdateExperimentTextViews(expID,expName,expDesc,expType,minTrials,region);
         trialManager.FB_FetchPublishedTrialCount(exp,currTrials);
 
+        //DK
+        final Button subscribe = findViewById(R.id.add_trial_button);
```
Comment (edenemilynan) [inline project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSearchedExperimentActivity.java]: Nope

File: project/app/src/main/res/layout/activity_view_subbed_experiment.xml [hunk 3]
```diff
@@ -111,38 +111,52 @@
...
+            android:layout_height="60dp"
+            android:text="Unsubscribe" />
+
+        <Button
+            android:id="@+id/view_data_button"
+            android:layout_width="150dp"
+            android:layout_height="60dp"
+            android:layout_marginStart="35dp"
+            android:layout_marginLeft="35dp"
+            android:onClick="dataButton"
```
Comment (edenemilynan) [inline project/app/src/main/res/layout/activity_view_subbed_experiment.xml]: You could remove this

File: project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSearchedExperimentActivity.java (hunk 2 above)
Comment (dkrammer) [reply project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSearchedExperimentActivity.java]: NOPE YOURSELF

**Review by edenemilynan — APPROVED** [review]
Thank you for making all those wee changes, sorry for the hassle :')

I mean I didn't do it but sorry on behalf of Android

**Pull request merged**